app/src/main/java/org/mozilla/fenix/wallpapers/WallpapersUseCases.kt

```diff
@@ -96,58 +96,78 @@ class WallpapersUseCases(
             // This should be cleaned up as improvements are made to the storage, file management,
             // and download utilities.
             withContext(Dispatchers.IO) {
-                val availableWallpapers = getAvailableWallpapers()
+                val availableWallpapers = possibleWallpapers.getAvailableWallpapers()
```

Comment:
follow-up nit: `getAvailableWallpapers` could be a good candidate for a unit test

Reply:
I personally prefer to avoid exposing functions individually for unit-testing. My philosophy is more along the lines of "you should unit-test the smallest public API". The functionality of that method is under test in several of the tests in `WallpaperUseCasesTest` already, like 
- `GIVEN wallpapers that expired WHEN invoking initialize use case THEN expired wallpapers are filtered out and cleaned up`
- `GIVEN wallpapers that expired and an expired one is selected WHEN invoking initialize use case THEN selected wallpaper is not filtered out`
- `GIVEN wallpapers that are in promotions outside of locale WHEN invoking initialize use case THEN promotional wallpapers are filtered out`

Reply:
I see the logic in this approach. As long as `getAvailableWallpapers` is only used in this one location, I agree it can stay as-is with your aforementioned tests.